docs/modules/ROOT/pages/general-info/general-info.adoc

```diff
@@ -1,138 +1,107 @@
+The Payara Platform Community Edition project is released each month. Each release incorporates fixes and enhancements from the Payara team, GlassFish upstream, and Enterprise only features.
 
-[[naming-strategy]]
-== Naming Strategy
+[[new-naming-strategy]]
+== New Naming Strategy (June 2020)
 
-The Payara Server naming strategy works off of the pre-existing GlassFish
-naming strategy: Append the year and quarter as the final dot version of
-the release. For example, for the Payara Server's release built on GlassFish
-4.1, released in quarter 3 of 2015, the version number would be payara-4.1.153.
+As of June 2020, The naming strategy of the Payara Platform Enterprise Edition releases is changed.
 
-In the case of updates, we will simply attach an additional point number
-to the end of the version number described above. For example, if a
-patch is released for Payara 4.1.152, the version number would be
-4.1.152.1. This will be in addition to any extra point releases that
-Oracle do for GlassFish, so it's possible for a version number to be
-something like 4.1.1.152.1!
+The general format of the version number is **Major Version Number** . **Minor Number** . **Patch Number**
 
-Releases made to date include:
+**Major Version** numbers will start from 5 (the current major version of Payara Platform) and only increment by 1 when a new major version (E.g Payara 6) is released.  Fundamental changes (e.g. Javax to Jakarta breaking change) will only be released in Major Versions. There is currently no set cadence for this version change. The change of a major version number is a product management and marketing decision.
 
-****
-2018 Q1 - Payara Server 5.181 +
-2018 Q1 - Payara Server 4.1.2.181 +
+Customers should expect changes to their applications and configuration scripts to move to a new Major version.
 
-2017 Q4 - Payara Server 4.1.2.174 +
-2017 Q3 - Payara Server 4.1.2.173 +
-2017 Q2 - Payara Server 4.1.2.172 +
-2017 Q1 - Payara Server 4.1.1.171 +
+**Minor Version** numbers will start from 20 (the year that Payara Enterprise and Community versions diverge)
+and increment by 1 quarterly.  These version changes will introduce significant features (typically originating from the Product Management process or stabilised and adopted from Community releases). 
 
-2016 Q4 - Payara Server 4.1.1.164 +
-2016 Q3 - Payara Server 4.1.1.163 +
-2016 Q2 - Payara Server 4.1.1.162 +
-2016 Q1 - Payara Server 4.1.1.161 +
+Features introduced during a minor version change must be backwards compatible with previous minor releases with no intervention required by an operator. 
 
-2015 Q4 - Payara Server 4.1.1.154 +
-2015 Q3 - Payara Server 4.1.153 +
-2015 Q2 - Payara Server 4.1.152 +
-2015 Q1 - Payara Server 4.1.151 +
+Features introduced during a minor version change can only introduce breaking changes as a last resort and with product management approval and appropriate customer messaging.
```

Comment:
```suggestion
Features introduced during a minor version change can only introduce breaking changes as a last resort and with a special caution. In that case, customers will be sufficiently informed about it and guided how to upgrade with minimal impact on their systems.
```

Again, no need to mention product management. The term "customer messaging" seems a bit alien to customers so I suggest a reworded sentence.